lrpar/src/lib/mf.rs

```diff
@@ -612,17 +624,19 @@ pub(crate) fn simplify_repairs(all_rprs: &mut Vec<Vec<ParseRepair>>)
     all_rprs.dedup();
 }
 
-pub(crate) struct Dist {
+pub(crate) struct Dist<StorageT> {
     terms_len: u32,
-    table: Vec<u32>
+    table: Vec<u32>,
+    phantom: PhantomData<StorageT>
 }
 
-impl Dist {
-    pub(crate) fn new<F>(grm: &YaccGrammar,
-                         sgraph: &StateGraph,
-                         stable: &StateTable,
-                         term_cost: F)
-                      -> Dist
+impl<StorageT: Hash + PrimInt + Unsigned> Dist<StorageT> {
+    pub(crate) fn new<F>
+                     (grm: &YaccGrammar<StorageT>,
+                      sgraph: &StateGraph<StorageT>,
+                      stable: &StateTable<StorageT>,
+                      term_cost: F)
+                   -> Dist<StorageT>
                    where F: Fn(TIdx) -> u8
     {
         // This is an extension of dist from the KimYi paper: it also takes into account reductions
```

Comment:
Should this comment be indented another couple of spaces?

Reply:
It's indented to 4 spaces already?

Reply:
I think you're right. For some reason it looks really wrong in GH.